test/event_unittest.py

```diff
@@ -26,14 +29,14 @@ def setUp(self):
         # Some configureation variables, by default, this is for the a handler
         self.config = {'vtn_ids': 'vtn_1,vtn_2,vtn_3,TH_VTN',
                        'ven_id': VEN_ID}
-        oadr_schema_file = open(os.path.join(SCHEMA_DIR, 'oadr_20a.xsd'))        # OpenADR
-        oadr_schema_doc = etree.parse(oadr_schema_file)
-        self.oadr_schema = etree.XMLSchema(oadr_schema_doc)
+        with open(os.path.join(SCHEMA_DIR, 'oadr_20a.xsd')) as oadr_schema_file:
+            oadr_schema_doc = etree.parse(oadr_schema_file)
+            self.oadr_schema = etree.XMLSchema(oadr_schema_doc)
         self.event_handler = event.EventHandler(**self.config)
         
         # Make things a little nicer for us to see
         print('')
-        print(40 * '=')
+        print((40 * '='))
```

Comment:
Why additional parenthesis? (in the other places in this PR i see you have them too)